website/docs/r/aviatrix_branch_router_tag.html.markdown

```diff
@@ -9,6 +9,8 @@ description: |-
 
 The **aviatrix_branch_router_tag** resource allows the creation and management of branch router config tags.
 
+~> **NOTE:** Applying this resource will immediately commit the config to the specified branch routers.
+
```

Comment:
~> **NOTE:** Creating this resource will automatically commit the config to the specified branch routes.